### PR TITLE
i18n(es): Update `cloudflare.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update `cloudflare.mdx` based on this commit 95e27f8f65d77406f130c807400af99af7775bb0

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 1]
```diff
@@ -169,6 +169,30 @@ export function GET({ params }) {
 }
 ```
 
+### `cloudflare.runtime`
+
+`runtime: "off" | "local" | "remote"`
+default `"off"`
+
+Esta bandera opcional habilita el servidor de desarrollo de Astro para poblar las variables de entorno y el objeto de solicitud de Cloudflare, evitando la necesidad de Wrangler.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:177]:
```suggestion
Esta bandera opcional permite que el servidor de desarrollo de Astro llene las variables de entorno y el objeto de solicitud de Cloudflare, evitando la necesidad de Wrangler.
```

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 2]
```diff
@@ -169,6 +169,30 @@ export function GET({ params }) {
...
+
+*   `local`: las variables de entorno están disponibles, pero el objeto de solicitud se rellena con un valor de marcador de posición estático.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:179]:
```suggestion
*   `local`: las variables de entorno están disponibles, pero el objeto de solicitud se rellena con un valor de placeholder estático. 
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**